lib/bulk/common.js

```diff
@@ -242,25 +251,30 @@ class BulkWriteResult {
 }
 
 /**
- * Create a new WriteConcernError instance (INTERNAL TYPE, do not instantiate directly)
- *
- * @class
- * @return {WriteConcernError} a WriteConcernError instance
+ * @classdesc An error representing a failure by the server to apply the requested write concern to the bulk operation.
  */
 class WriteConcernError {
+  /**
+   * Create a new WriteConcernError instance (INTERNAL TYPE, do not instantiate directly)
+   *
+   * @class
+   * @return {WriteConcernError} a WriteConcernError instance
```

Comment:
we don't document return values for most constructors, should we be doing that here?